Sources/TuistGenerator/Generator/ConfigGenerator.swift

```diff
@@ -291,6 +291,10 @@ final class ConfigGenerator: ConfigGenerating {
             if devices.contains(.ipad), devices.contains(.mac) {
                 settings["SUPPORTS_MACCATALYST"] = "YES"
                 settings["DERIVE_MACCATALYST_PRODUCT_BUNDLE_IDENTIFIER"] = "YES"
+            } else {
+                // Unless explicitly specified, when the platform the Product is a framework, these default to YES.
+                settings["SUPPORTS_MACCATALYST"] = "NO"
```

Comment:
This will cause the setting to always be set. But if the desired setting matches the default (`NO` for app, `YES` for framework) setting it is unnecessary.

Reply:
If a framework target specifies iPad and Mac, then the desired behavior matches the default Xcode behavior so there's no need to set `SUPPORTS_MACCATALYST`.

Reply:
I would still set it explicitly instead of relying on Xcode default (which was also the old logic), so I would leave it as @TheInkedEngineer implemented it

Reply:
Fair enough.